src/app/assets/drag-indicator-grey.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I assume that Nick D supplies you with these? If so, just need to make sure that he has either created them or sourced them from somewhere that allows us to use/distribute them.

Reply:
It's the same icon as the drag handle on the sort authors list but in different colour